publish.js

```diff
@@ -10,20 +10,24 @@ You may obtain a copy of the License at
 https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
 */
 
+/* eslint-env node */
+
 "use strict";
 
 var publish = {};
 var path = require("path");
 var extend = require("extend");
 
+// TODO: The supported version of node.js does not yet support ES6 template strings
+// When version node.js 4.x.x is supported this can be replaced by native support.
+var es6Template = require("es6-template-strings");
```

Comment:
@amb26 I agree we should be able to set the minimum to 4.0.0. However, while exploring ES6 template literals. ( https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals ) it is required that the literal ``use ${backTicks}``. These back ticks fail our JSON linting. I can't actually find anywhere that indicates that they should be valid or not in JSON. Any thoughts?